src/main/java/nextstep/subway/fare/domain/Fare.java

```diff
@@ -0,0 +1,93 @@
+package nextstep.subway.fare.domain;
+
+import nextstep.subway.fare.exception.BelowZeroIntegerException;
+import nextstep.subway.fare.exception.LessZeroOperationException;
+
+import javax.persistence.Column;
+import javax.persistence.Embeddable;
+import java.math.BigDecimal;
+import java.util.Objects;
+
+import static java.math.BigDecimal.ZERO;
+import static java.math.BigDecimal.valueOf;
+
+@Embeddable
+public class Fare {
+    public static final String NAME = "요금";
+    public static final Fare DEFAULT_FARE = new Fare(1250);
+
+    @Column(name = "fare")
+    private BigDecimal value = new BigDecimal(0);
```

Comment:
`Fare` 객체로 잘 포장해주셨네요 👍 

```suggestion
    @Column(name = "fare")
    private BigDecimal value = BigDecimal.ZERO;
```